docs/web-component-samples/index.html

```diff
@@ -4,23 +4,61 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>CC Widgets as web-component</title>
+    <style>
+      .disabled {
+        display: none;
+      }
+    </style>
   </head>
   <body>
     <h1>Contact Center widgets as web-component</h1>
+    <input
+      type="text"
+      placeholder="Enter your access token"
+      id="access_token_elem"
+      onkeyup="switchButtonState()"
+    />
+    <button onclick="initWidgets()" disabled>Init Widgets</button>
     <script src="dist/bundle.js"></script>
-    <widget-cc-station-login></widget-cc-station-login>
-    <widget-cc-user-state></widget-cc-user-state>
+    <div id="widgets-container" class="disabled">
+      <widget-cc-station-login id="cc-station-login"></widget-cc-station-login>
+      <widget-cc-user-state></widget-cc-user-state>
+    </div>
   </body>
   <script>
-    const token = '';
-    
-    const webexConfig = {
-        fedramp: false,
-        logger: {
-        level: 'log'  // TODO: We will add more logging levels later and set the righ levels
-        },
+    const widgetsContainer = document.getElementById('widgets-container');
```

Comment:
This file will keep increasing so does it make sense to create separate .js file and have that added in this script section

Reply:
We have an index.js file that is importing the widgets, maybe we can move this code in that file.

Reply:
Done.